app/javascript/components/RemoveButton.tsx

```diff
@@ -1,14 +1,9 @@
-import cx from "classnames";
 import * as React from "react";
 
 // This component requires a `position: relative;` somewhere up
 // the chain for correct top/right positioning.
-export const RemoveButton = ({ className, style, ...props }: React.HTMLAttributes<HTMLDivElement>) => (
-  <div
-    {...props}
-    className={cx("remove-button", className)}
-    style={{ ...style, width: "var(--big-icon-size)", display: "inline-block", cursor: "pointer" }}
-  >
+export const RemoveButton = ({ style, ...props }: React.HTMLAttributes<HTMLDivElement>) => (
+  <div {...props} style={{ ...style, width: "var(--big-icon-size)", display: "inline-block", cursor: "pointer" }}>
```

Comment:
Removed the `remove-button` class here and in tests as it doesn't do anything anymore